src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/drive/Drivetrain.kt

```diff
@@ -164,7 +164,7 @@ class Drivetrain(val gyroIO: GyroIO, swerveModuleIOs: DrivetrainIO) : SubsystemB
   var omegaVelocity = 0.0.radians.perSecond
 
   override fun periodic() {
-    val startTime = Clock.realTimestamp
+    val startTime = AdvantageClock.realTimestamp
```

Comment:
why is this `AdvantageClock`? I feel like it should just go in `Clock`, and you should actually rename `FPGATimestamp` to something else (it's not the real FPGA timestamp which is the whole point of `getRealTimestamp()`

Reply:
not sure why this is in this commit--ig i made the change here and didn't know,, it is like this in `superstructure` where the change was actually made